src/Components/People.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class People extends Component {
+    constructor() {
+        super()
+        this.state = {
+            textInput: '',
+            people: [],
+            currentPerson: [],
+        };
+    };
+
+    handleChange = (e) => {
+        this.setState({ textInput: e.target.value });
+    };
+
+    handleSubmit = async (e) => {
+        e.preventDefault();
+        const { textInput, people} = this.state;
+        let selectedPerson
+
+        for(let person of people) {
```

Comment:
This would be a great time to use JavaScript's built-in `.find` method on arrays. (And if you don't know it, this would be a great time to add it to your toolbox!) It will return you the element you want, just like `.filter`, but with only one element returned instead of a whole array.